internal/services/sast-metadata_test.go

```diff
@@ -0,0 +1,78 @@
+package services
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/checkmarx/ast-cli/internal/wrappers"
+	"github.com/checkmarx/ast-cli/internal/wrappers/mock"
+	"github.com/stretchr/testify/assert" //nolint:depguard
+)
+
+func TestGetSastMetadataByIDs(t *testing.T) {
+	tests := []struct {
+		name    string
+		scanIDs []string
+	}{
+		{
+			name:    "Multiple batches",
```

Comment:
200 will not be multiple right? because of the -1 . can you do it 2200 instead? that it'll reach 10

Reply:
Changed it to 5000